frontend/package.json

```
@@ -61,6 +66,19 @@
       "last 1 safari version"
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx,ts,tsx}": [
+      "lint --fix"
```

This one also didn't work here because there's no lint executable. Does this run in yours?

```suggestion
      "npm run lint -- --fix"
```